OpenIPSL/package.mo

```diff
@@ -4,7 +4,7 @@ package OpenIPSL "Open-Instance Power System Library"
 
 annotation (preferredView="info",
   Protection(access = Access.packageDuplicate),
-  uses(Complex(version="3.2.3"), Modelica(version="3.2.3")),
+  uses(Complex(version="3.2.3"), Modelica(version="3.2.3"), Modelica_Synchronous(version="0.93.0")),
   version="2.0.0-beta.1",
```

Comment:
Should we update version="2.0.0" ?

Reply:
I was planning to do those version and date bumps separately but since I guess I will squash this PR anyway I might just as well do it right here.